doc/interop-test-descriptions.md

```diff
@@ -1057,6 +1069,14 @@ This test must be configurable via a few different command line flags:
 * `soak_min_time_ms_between_rpcs`: The minimum time in milliseconds between
   consecutive RPCs. Useful for limiting QPS.
 
+* `soak_num_threads`: Specifies the number of threads to use for concurrently 
+  executing the soak test. Each thread performs `soak_iterations / soak_num_threads`
+  RPCs.
+
+This value defaults to 1 (i.e., no concurrency) but can be 
+  increased for concurrent execution. The total soak_iterations must be 
+  divisible by soak_num_threads.
```

Comment:
Or what? "The client can fail if not evenly divisible"?

Reply:
" if (soakIterations % numThreads != 0) {
      throw new IllegalArgumentException("soakIterations must be evenly divisible by numThreads.");
    }"

Reply:
Have the text say what can happen. It sounds like the answer is, "The client should fail if soak_iterations is not evenly divisible by soak_num_threads."

Reply:
I think you're making a good point! It will be clearer to users if I clarify the behavior—'The client should fail if soak_iterations is not evenly divisible by soak_num_threads'—in the documentation. Updated.